cpp/oneapi/dal/algo/knn/common.hpp

```diff
@@ -98,8 +98,14 @@ ONEDAL_EXPORT result_option_id get_responses_id();
 /// what should algorithm return
 namespace result_options {
 
+/// Signs that algorithm must return nearest neighbors indices
 const inline result_option_id indices = detail::get_indices_id();
+
+/// Signs that algorithm must return nearest neighbors distances
 const inline result_option_id distances = detail::get_distances_id();
+
+/// Signs that algorithm must return results of :expr<classification>
+/// **Note:** This result is prohibited in :expr<search> task
```

Comment:
Are you sure that `prohibited` is the best word choice here? Classification results are simply not available when you run this task, right? I think it's better to say that doing this will result in an error.

Reply:
```suggestion
/// **Note:** This result is not available if :expr<task> is :expr<search>.
```

Reply:
Is it good enough?

Reply:
```suggestion
/// **Note:** This result is not available for the :expr<search> task.
```